sdk/core/azure-core/test/ut/CMakeLists.txt

```diff
@@ -117,7 +117,8 @@ if(BUILD_TRANSPORT_CURL)
 
   if (MSVC)
     # warning C4389: '==': signed/unsigned mismatch
-    target_compile_options(azure-core-libcurl-test PUBLIC /wd4389)
+    # # - C6323: Google comparisons 
+    target_compile_options(azure-core-libcurl-test PUBLIC /wd4389 /wd6323)
```

Comment:
Isn't this the wrong warning? I thought it was C6326.

https://docs.microsoft.com/en-us/cpp/code-quality/c6323?view=msvc-160
VS
https://docs.microsoft.com/en-us/cpp/code-quality/c6326?view=msvc-160